config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -94,6 +103,9 @@ periodics:
         requests:
           memory: "9000Mi"
           cpu: 2000m
+  annotations:
+    test_tab-name: kubeadm-kinder-upgrade-1.12-1.13
+    testgrid-dashboards: sig-release-1.13-all
```

Comment:
for *kind(er)* jobs it should be possible to enable `alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com` on the job side too.